### PR TITLE
Navbar stays on Dashboard for Test Patient List

### PR DESCRIPTION
**Pull request by dczulada**
Previously the navbar highlighted the Master Patient List
screen shot 2016-01-28 at 4 07 15 pm

### CODE REVIEW

**Comment by thecristen** [comment]
Good catch!

File: app/views/application/_navbar.html.erb [hunk 1]
```diff
@@ -9,13 +9,13 @@
     <div class="collapse navbar-collapse" id="cypressNavbar">
       <ul class="nav navbar-nav navbar-right">
         <% if user_signed_in? %>
-          <li <% if params[:controller] != 'records' && params[:controller] != 'registrations' %>class="active"<% end %>>
+          <li <% if (params[:controller] == 'records' && params[:product_test_id]) || (params[:controller] != 'records' && params[:controller] != 'registrations') %>class="active"<% end %>>
```
Comment (jaebird123) [inline app/views/application/_navbar.html.erb]: maybe the long conditional could go in a helper function

Comment (ssayer) [inline app/views/application/_navbar.html.erb]: you should be using the rails routes, not looking at params to figure out username controller

Comment (okeefm) [inline app/views/application/_navbar.html.erb]: http://stackoverflow.com/a/4354547 is a decent way to use the Rails routes to do just this, e.g. `current_page?(:some_route)`

**Pull request merged**